rust/xaynet-analytics/src/database/screen_route/data_model.rs

```diff
@@ -47,3 +47,44 @@ impl From<ScreenRoute> for RelationalField {
         }
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    #[test]
+    fn test_screen_route_try_from_adapter() {
+        let timestamp_str = "2021-01-01T01:01:00+00:00";
+        let timestamp_parsed = DateTime::parse_from_rfc3339(timestamp_str)
+            .unwrap()
+            .with_timezone(&Utc);
+        let screen_route = ScreenRoute::new("route", timestamp_parsed);
+        let adapter = ScreenRouteAdapter::new("route", timestamp_str);
+        assert_eq!(ScreenRoute::try_from(adapter).unwrap(), screen_route);
+    }
+
+    #[test]
+    fn test_adapter_try_into_screen_route() {
+        let timestamp_str = "2021-01-01T01:01:00+00:00";
+        let timestamp_parsed = DateTime::parse_from_rfc3339(timestamp_str)
+            .unwrap()
+            .with_timezone(&Utc);
+        let screen_route = ScreenRoute::new("route", timestamp_parsed);
+        let adapter = ScreenRouteAdapter::new("route", timestamp_str);
+        assert_eq!(Into::<ScreenRouteAdapter>::into(screen_route), adapter);
+    }
+
+    #[test]
+    fn test_screen_route_from_relational_field() {
+        let timestamp_str = "2021-01-01T01:01:00+00:00";
+        let timestamp_parsed = DateTime::parse_from_rfc3339(timestamp_str)
+            .unwrap()
+            .with_timezone(&Utc);
+        let screen_route = ScreenRoute::new("route", timestamp_parsed);
+        let relationa_field = RelationalField {
```

Comment:
```suggestion
        let relational_field = RelationalField {
```
similarly in other data models

Reply:
thanks, done